packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -78,9 +78,10 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
    * For some components which do not strictly follow the order of the array, we might
    * re-render the initial tiles -> dispose them -> create new tiles, we need to take care of
    * this case when those components are here
+   * Note: If for some reason the current number of videos rendered is more than the maximum allowed, 4 videos will be rendered
    */
   const [indexesToRender, setIndexesToRender] = useState<number[]>([
-    ...Array(maxRemoteVideoStreams - activeVideoStreams).keys()
+    ...Array(Math.max(maxRemoteVideoStreams - activeVideoStreams, 4)).keys()
```

Comment:
This is only the initial value of the indices of overflow gallery to render. The array of indices will change to the indices of the first page of the overflow gallery when the overflow gallery is rendered. I think this is not needed.

Reply:
Tested and works well without initialization, I will push the code soon to address this